popup.js

```diff
@@ -35,3 +37,39 @@ function updateDistance(distance) {
     </div>
   `);
 }
+// add current website to list of paused websites
+document.getElementById('pause-current-site')?.addEventListener('click', () => {
```

Comment:
suggestion: we have `jquery` so here you can use jquery selector vs document.getElementById
```suggestion
const pauseBtn = $("pause")
pauseBtn.addEventListener('click', () => {
```